src/main/java/com/demoqa/shop/pages/HomePage.java

```diff
@@ -7,24 +7,36 @@
 
 public class HomePage extends BasePage {
 
-
-    private static WebElement element = null;
     @FindBy(how = How.CLASS_NAME, using = "woocommerce-store-notice__dismiss-link")
     public WebElement lnk_Dismiss;
     @FindBy(how = How.XPATH, using = "//a[contains(text(),'My Account')]")
     public WebElement lnk_MyAccount;
+    @FindBy(how = How.XPATH, using = "//a[contains(text(),'My Wishlist')]")
+    public WebElement lnk_MyWishlist;
+    @FindBy(how = How.XPATH, using = "//a[contains(text(),'Checkout')]")
+    public WebElement lnk_Checkout;
+    @FindBy(how = How.XPATH, using = "//span[contains(@class,'cart-name-and-total')]")
+    public WebElement lnk_Cart;
+    @FindBy(how = How.XPATH, using = "//a[contains(@class,'noo-search')]")
+    public WebElement lnk_Search;
 
     public HomePage(WebDriver driver) {
```

Comment:
here we have public method, maybe would be better to keep lnk_Dismiss public

Reply:
lnk_Dismiss is  private now